docs/pages/getting-started/providers/cognito.mdx

```diff
@@ -83,6 +84,18 @@ app.use(
   </Code.Express>
 </Code>
 
+### Options
+
+The **Amazon Cognito Provider** comes with a set of default options:
```

Comment:
Although this whole part about the options just links to the code for the provider, this is kind of implied.. 

Let's get rid of this part and keep the Callout about the hosted domain settings as well as the link for hosted domain above, thanks!